.circleci/config.yml

```diff
@@ -31,6 +31,20 @@ jobs:
           paths:
             - api/node_modules
 
+  web dependency vulnerability scan:
+    docker:
+      - image: node:8
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-cms-hitech-apd-{{ checksum "web/package-lock.json" }}
+      - run:
+          name: vulnerability scan
+          working_directory: ~/project/web
+          command: |
+            npx snyk auth $SNYK_TOKEN
```

Comment:
Right now this token is tied to my account, but the token is set in the CircleCI project settings so we can _*easily*_ switch it to someone else later with no code changes.  😄